Svistelnikoff@work 03

svistelnikoff opened the pull request.

svistelnikoff committed:
03-Service: Add get/set service implementation

Signed-off-by: Vladimir Svistelnikov <svistelnikoff@gmail.com>

svistelnikoff committed:
03-Service: Add set application implementation

Signed-off-by: Vladimir Svistelnikov <svistelnikoff@gmail.com>

svistelnikoff committed:
03-Service: Add get application implementation

Signed-off-by: Vladimir Svistelnikov <svistelnikoff@gmail.com>

svistelnikoff committed:
03-Service: Android build system integration

Signed-off-by: Vladimir Svistelnikov <svistelnikoff@gmail.com>

AksonovSergei reviewed (approved):
Good job!
You did more then it was requested. I mean using of intent to set/get value.

svistelnikoff committed:
apps: implement HAL-related functionality

- add UI controls
- add HAL-service IPC
- build fixes
- minor improvements, code prettyfy

Signed-off-by: Vladimir Svistelnikov <svistelnikoff@gmail.com>
